osu.Game/Screens/Ranking/Statistics/StatisticsPanel.cs

```diff
@@ -76,6 +76,8 @@ private void populateStatistics(ValueChangedEvent<ScoreInfo> score)
 
             if (newScore.HitEvents == null || newScore.HitEvents.Count == 0)
             {
+                spinner.Hide();
```

Comment:
This needs to be hidden when `newScore` is `null` (see conditional directly above).

Safest is to always hide outside of the conditionals altogether (before line 74), I think.